GoogleForCreators/web-stories-wp
# Fix panel title colours and focus

## Context
With this PR the panel title colours differ when collapsed/expanded.

## To-do



## User-facing changes
| Before | After |
|--------|------|
| ![before](https://user-images.githubusercontent.com/3294597/111590591-95b6ed80-87c6-11eb-8372-6d4df7de45b7.gif) | ![title-after](https://user-images.githubusercontent.com/3294597/111590248-2640fe00-87c6-11eb-9521-f0f2812453fb.gif) |
| <img width="305" alt="Screenshot 2021-03-18 at 09 47 06" src="https://user-images.githubusercontent.com/3294597/111590569-8d5eb280-87c6-11eb-863b-cefc5a130bfc.png"> | <img width="316" alt="Screenshot 2021-03-18 at 09 44 59" src="https://user-images.githubusercontent.com/3294597/111590327-407adc00-87c6-11eb-88ae-cb9391b3af5b.png"> |




### QA






This PR can be tested by following these steps:

1. Expand and collapse the design panels.
2. Verify the title colour changes to lighter when expanded and to darker when collapsed.
3. Tab into a panel header and verify the focus line around the headline looks as expected, too.

See [Figma.](https://www.figma.com/file/bMhG3KyrJF8vIAODgmbeqT/Design-System?node-id=3389%3A38563)

### UAT







This PR can be tested by following these steps:

1.

## Checklist




---



Fixes #6815
